src/auro-hyperlink.js

```diff
@@ -110,7 +110,7 @@ export class AuroHyperlink extends ComponentBase {
         href="${ifDefined(this.role ? undefined : this.safeUri)}"
         rel="${ifDefined(this.target || this.rel ? this.getRelType(this.target, this.rel) : undefined)}"
         referrerpolicy="${ifDefined(this.referrerpolicy ? this.defaultReferrerPolicy : undefined)}"
-        role="${ifDefined(this.role === 'button' ? this.role : undefined)}"
+        role="${ifDefined(this.ariarole === 'button' ? this.role : undefined)}"
```

Comment:
I don't think this is the correct way to achieve what you want here but I'm not 100% sure with what the intention is.

The rest of our checks in here all use `this.role`, with `this.ariarole` not actually being a thing in the Hyperlink or the ComponentBase class 🤔 So if the intention is to use `ariarole` to affect the root anchor tag instead, we would need to actually create ariaRole on this component since we don't tell Lit to care about this value as far as I'm aware.